### PR TITLE
newsletter section was added

### PR DESCRIPTION
**Pull request by anogire**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h2 class="title-light">Newsletter</h2>
           </div>
           <form class="col-md-6">
             <div class="input-wrap">
-              <input class="input-main input-main--email" type="email" name="email" placeholder="Your email address"
+              <input class="input-main input-main--email" type="email" name="email" placeholder="your email address"
```
Comment (Plakhotenko) [inline index.html:27]: Why do need this changes? According to design it should be "Your" not "your"

**Review by Plakhotenko — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Plakhotenko) [reply index.html:27]: Looks good. You can merge it as soon as you're ready.

**Comment by Plakhotenko** [comment]
Looks good. You can merge it as soon as you're ready.

File: index.html (hunk 1 above)
Comment (anogire) [reply index.html:27]: for pull request only

**Pull request merged**